Dockerfile

```diff
@@ -0,0 +1,72 @@
+#################################################################################
+# Step to build the application assets
+#################################################################################
+FROM node:12 AS build
+RUN mkdir -p /app
+WORKDIR /app
+
+# Config build args
+ARG PUBLIC_URL=/
```

Comment:
@rrequero why do we have all these definitions in here? some of them are ok, but the ones that are environment dependant like `REACT_APP_MAP_URL`, `REACT_APP_V2_API_ENDPOINT`, `WORKSPACE_API_URL`, `REACT_APP_GOOGLE_TAG_MANAGER_KEY`, `REACT_APP_SHARE_BASE_URL`, `DEFAULT_WORKSPACE`, should be in cloud build level? Actually I would move every variable there, just to have consistency.

Reply:
@enriquetuya these are the args that cloudbuild replaces with the environment variables of the cloudbuild task when it generates the images. I configured by default these values.